layouts/shortcodes/observability_pipelines/prerequisites/kafka.md

```diff
@@ -0,0 +1,4 @@
+To use Observability Pipelines's Kafka source, you need the following information available:
+
+- The hosts and ports of the Kafka bootstrap servers, which clients should use to connect to the Kafka cluster and discovers all the other hosts in the cluster.
```

Comment:
```suggestion
- The hosts and ports of the Kafka bootstrap servers, which clients should use to connect to the Kafka cluster and discover all the other hosts in the cluster.
```
I think the client is still the subject of "discover," if not you might reword instead.